src/GrumPHP/Console/Helper/PathsHelper.php

```diff
@@ -206,8 +204,7 @@ public function getBinCommand($command, $forceUnix = false)
      */
     public function getRelativePath($path)
     {
-        $realpath = $this->getAbsolutePath($path);
-        return $this->fileSystem->makePathRelative($realpath, $this->getWorkingDir());
+        return $this->getAbsolutePath($path);
```

Comment:
Kinda makes this method obsolete, will refactor it out if absolute paths are the way forward